app/Http/Requests/Service/UpdateRequest.php

```diff
@@ -280,6 +281,16 @@ public function rules()
                 new RootTaxonomyIs(Taxonomy::NAME_CATEGORY),
             ],
 
+            'eligibility_types' => $this->serviceEligibilityRules(),
+            'eligibility_types.taxonomies' => ['array'],
+            'eligibility_types.taxonomies.*' => [
+                'uuid',
+                'exists:taxonomies,id',
+                new RootTaxonomyIs(Taxonomy::NAME_SERVICE_ELIGIBILITY),
+            ],
+            'eligibility_types.custom' => ['array'],
+            'eligibility_types.custom.*' => ['nullable', 'string', 'min:1', 'max:255'],
+
```

Comment:
As we will only accept specific keys on the `eligibility_types.custom` object, it might be better to set them out specifically:

```suggestion
'eligibility_types.custom' => ['array'],
'eligibility_types.age_group' => ['nullable', 'string', 'min:1', 'max:255'],
'eligibility_types.disability' => ['nullable', 'string', 'min:1', 'max:255'],
'eligibility_types.employment' => ['nullable', 'string', 'min:1', 'max:255'],
'eligibility_types.gender' => ['nullable', 'string', 'min:1', 'max:255'],
'eligibility_types.housing' => ['nullable', 'string', 'min:1', 'max:255'],
'eligibility_types.income' => ['nullable', 'string', 'min:1', 'max:255'],
'eligibility_types.language' => ['nullable', 'string', 'min:1', 'max:255'],
```

Reply:
Done.